joomla/joomla-cms
# [4.2] Installation updateuserids

### Summary of Changes
during the installation we run various queries to update default data with the superuseerid created during the install


### Testing Instructions
Clean install from the generated full package
Check that you never get an error message 
![image](https://user-images.githubusercontent.com/1296369/162768476-82d2878a-71b1-4d0c-89ad-78203352c4ce.png)



### Actual result BEFORE applying this Pull Request
Running queries on tables that have no data 


### Expected result AFTER applying this Pull Request
No change to the generated install


### Documentation Changes Required
none
